### PR TITLE
Docs for typed endpoint functions and `CommentBody` functions

### PR DESCRIPTION
**Pull request by CTNicholas**
Docs API reference for:
-  https://github.com/liveblocks/liveblocks/pull/1283
- https://github.com/liveblocks/liveblocks/pull/1301

**TODO**
- [x] Rooms
- [x] Storage
- [x] Yjs
- [x] Schema validation
- [x] `CommentBody` functions
- [x] Update email notification guide
- [ ] Check for any API changes in PRs before merging

### CODE REVIEW

**Comment by nimeshnayaju** [comment]
Thank you for doing this @CTNicholas !!!

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 1]
```diff
@@ -273,7 +275,681 @@ export default async function auth(req, res) {
...
+  const { data, nextPage } = await liveblocks.getRooms({ startingAfter });
+  pages.push(data);
+  startingAfter = nextPage;
+}
+```
+
+#### Liveblocks.createRoom [#post-rooms]
+
+Programmatically creates a new room from a room ID. The `defaultAccesses` option
+is required—setting it to `["room:write"]` creates a public room, whereas
```
Comment (nimeshnayaju) [inline docs/pages/api-reference/liveblocks-node.mdx:347]: Should the `-` have a space before and after?

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 2]
```diff
@@ -273,7 +275,681 @@ export default async function auth(req, res) {
...
+```
+
+Group and user permissions are only used with
+[ID token authorization](/docs/api-reference/liveblocks-node#id-tokens), learn
+more about
+[managing permission with ID tokens](/docs/rooms/permissions/id-token).
+
+#### Liveblocks.getRoom [#get-rooms-roomId]
+
+Returns a room. Throws an error if the room isn’t found. This is a wrapper
```
Comment (nimeshnayaju) [inline docs/pages/api-reference/liveblocks-node.mdx:396]: We throw our custom error called `LiveblocksError` if the room isn’t found. Should we mention this error class too somewhere?

```ts
if (!res.ok) {
  const text = await res.text();
  throw new LiveblocksError(res.status, text);
}
```

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 3]
```diff
@@ -273,7 +275,681 @@ export default async function auth(req, res) {
...
+};
+```
+
+```ts highlight="4"
+import { UserMeta } from "../liveblocks.config";
+
+const activeUsers =
+  await liveblocks.getActiveUsers<UserMeta["info"]>("my-room-id");
+
+// [{ ..., info: { name: "Nimesh", avatar: "https://..." }, ...]
```
Comment (nimeshnayaju) [inline docs/pages/api-reference/liveblocks-node.mdx:503]: 🫡

File: docs/pages/api-reference/liveblocks-node.mdx (hunk 2 above)
Comment (CTNicholas) [reply docs/pages/api-reference/liveblocks-node.mdx:396]: Yes, good idea! Could you write a code snippet showing me the recommended way to display and type an error?

File: docs/pages/api-reference/liveblocks-node.mdx (hunk 1 above)
Comment (CTNicholas) [reply docs/pages/api-reference/liveblocks-node.mdx:347]: This an em dash, and they're mostly used [without surrounding spaces](https://practicaltypography.com/hyphens-and-dashes.html#:~:text=Em%20and%20en%20dashes%20are%20typ%C2%ADi%C2%ADcally%20set%20flush%20against%20the%20sur%C2%ADround%C2%ADing%20text.%20Some%20fonts%20in%C2%ADclude%20a%20lit%C2%ADtle%20white%20space%20around%20the%20em%20dash%3B%20some%20don%E2%80%99t.%20If%20your%20em%20dashes%20look%20like%20they%E2%80%99re%20be%C2%ADing%20crushed%E2%80%94par%C2%ADtic%C2%ADu%C2%ADlarly%20if%20you%E2%80%99re%20set%C2%ADting%20type%20on%20screen%E2%80%94it%E2%80%99s%20fine%20to%20add%20word%20spaces%20be%C2%ADfore%20and%20after.), but you can add them if your text feels crushed. I agree, this doesn't look right here, I'll change it!

If you're interested in Typography, or would like to check best practices when writing, I super recommend [Practical Typography](https://practicaltypography.com/). I always try to follow their advice 🙂

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 4]
```diff
@@ -324,15 +1000,243 @@ returns the same response.
...
+  commentId: "my-comment-id",
 });
 ```
 
 ---
 
-## WebhookHandler [#WebhookHandler]
+## Error handling [#Error-handling]
+
+Errors in our API methods, such as network failures, invalid arguments, or
```
Comment (nimeshnayaju) [inline docs/pages/api-reference/liveblocks-node.mdx:1013]: @marcbouchenoire Added a section for error handling. Do you mind taking a look when are free?

**Review by nimeshnayaju — APPROVED** [review]

File: docs/pages/api-reference/liveblocks-node.mdx (hunk 4 above)
Comment (marcbouchenoire) [reply docs/pages/api-reference/liveblocks-node.mdx:1013]: Nice! Great addition, I made a few tiny tweaks and I think it's ready to go.

**Review by marcbouchenoire — APPROVED** [review]

**Pull request merged**